src/paddle.ts

```diff
@@ -8,6 +8,9 @@ export enum SubscriptionCycles {
 export const isPlusMember = (cycle: SubscriptionCycles | undefined): boolean =>
   !!cycle?.length || false;
 
+export const plusMemberSince = (createdAt?: Date | string): Date | null =>
+  !!createdAt ? new Date(createdAt) : null;
```

Comment:
we have `transformDate`, maybe you can use that?

Reply:
Yeah, can use that. Moved `transformDate` into `date.ts` for common use. Lets just hope it doesn't throw a infinite loop :triggered: